src/main/java/frameworksdrivers/Gateways/StudentDbGateway.java

```diff
@@ -106,48 +106,52 @@ public void saveUser(CreateStudentDsModel student) {
         try{
             Map<String, Float> courses = student.getCourseList();
             if (courses.isEmpty()){
-               saveUserNoCourses(student);}
-            else {
-            for (Map.Entry<String, Float> course : courses.entrySet()) {
-                String SQL = "INSERT INTO " + DATABASE_NAME +
-                        " (StudentID, Password, CourseID, CourseGrade) VALUES (?, ?, ?, ?)";
-                PreparedStatement statement = connection.prepareStatement(SQL);
-                statement.setString(1, student.getUsername());
-                statement.setString(2, student.getPassword());
-                statement.setString(3, course.getKey());
-                statement.setFloat(4, 0.0f);
-                statement.executeUpdate();}
```

Comment:
I indented this to conform with the if else, and the for loop to improve readability.